backend/db/migrations/R__0.21.0_CE-789.sql

```diff
@@ -0,0 +1 @@
+update  public.geo_organization_unit_code set administrative_office_ind = true where geo_organization_unit_code='COSHQ';
```

Comment:
This doesn't need a new Script.   It can go in R__Create-Test-Data in order to ensure that the code table version will be updated.

when redeploying we probably want to delete the PVC on openshift dev as Flyway doesn't like it when scripts are removed.